### ISSUE TITLE
Does 389ds use Java?

### ISSUE THREAD

**Issue by dreua** [389ds/389-ds-base#5069]
The [389 Directory Server Wikipedia page](https://en.wikipedia.org/wiki/389_Directory_Server) of most languages lists 389ds as "Written in C, Python, Java, Perl, shell script". I don't believe Java should be in this list, maybe someone confused it with JavaScript?

(This question arose from Log4Shell incident. If we could confirm 389ds does not use Java that would make sure it's not affected.)

Once this is cleared up, I can fix the English and German Wikipedia pages if you want me to.

**Comment by dreua** [comment]
I found one java file, but that doesn't qualify as "written in" imo: https://github.com/389ds/389-ds-base/blob/master/ldap/servers/slapd/test-plugins/clients/ReqExtOp.java (And it doesn't use log4j :wink: )

**Comment by mreynolds389** [comment]
Well in RHEL 7 we used to have a java console (389-admin, 389-admunutil, and idm-console-framework).  This was all in java as a standalone accompaniment product, and when I checked those admin packages they were not using log4j anyway.  Starting in RHEL 8 (389-ds-base-1.4.x) we stopped using the old java console, and created a Cockpit plugin as the frontend.

Our core server, 389-ds-base, it written in C, and most of our CLI tools are now in python (except for logconv.pl, but there are plans to port that to python someday).  Since we no longer maintain 389-admin & friends we no longer directly use java in any capacity.  So the wiki is definitely outdated, and java should definitely be removed.  And Shell Scripts, we don't use them anymore either.

If you don't mind updating the wiki that would be great!
